CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+#1.3.0
+* Redis Cache: storing searches in redis with smart refresh
```

Comment:
One more thing: we need to mark this as experimental because we intend to remove it in the next version when we figure out a better API to inject a caching strategy (so this repo isn't tightly coupled to redis).

Normally, we'd require documentation on this in the readme, but since this is going to be removed shortly we can skip that step as long as we clearly indicate that it's experimental and likely to be removed.